rules/windows/registry/registry_event/registry_event_malware_qakbot_registry.yml

```diff
@@ -0,0 +1,21 @@
+title: Potential Qakbot Registry Activity
+id: 1c8e96cd-2bed-487d-9de0-b46c90cade56
+status: experimental
+description: Detects registry key used by IceID in distributing malicious OneNote campaign
+references:
+    - https://www.zscaler.com/blogs/security-research/onenote-growing-threat-malware-distribution
+author: Hieu Tran
+date: 2023/03/13
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    selection:
+        TargetObject|endswith: '\Software\firm\soft'
```

Comment:
I'm not a fan, as this path is very specific to this report, and is not monitored in sysmon configurations such as sysmon-modular,SwiftOnSecurity...

Reply:
I would even add it as `\Software\firm\soft\name` or at least as `\Software\firm\soft\` to be more specific. The issue with the sysmon-config is true, but shouldn't be a reason not to include it so Sigma. 

I would also increase the score to high or critical, if the registry path is specific enough.

Reply:
- These are malware rules so it's okay for them to be specific. 
- These will be transferred to the DFIR folder in the future
- The key is specific as it's described in the report I will increase to high and see.